templates/projects/tasks_browse.webpack.ejs

```diff
@@ -542,6 +545,28 @@
   </div>
 </div>
 
+<!-- Update assign worker -->
+<div class="modal fade" id="update-assign-worker-modal" tabindex="-1" role="dialog" aria-labelledby="assign-worker-modal-label">
+
+  <div class="modal-dialog modal-sm" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">Assign Workers to Task</h4>
+      </div>
+      <div class="modal-body">
+
+        <assign-worker ref="assignworker"></assign-worker>
+
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-primary" id="save-assign-worker-modal" @click="$refs.assignworker.save()">Update</button>
```

Comment:
Update button can be disabled initially and get enabled only with any changes to assign/remove user list?

Reply:
I'm not sure if I can finish this by the deadline